eras/shelley-ma/impl/cardano-ledger-shelley-ma.cabal

```diff
@@ -64,7 +64,6 @@ library
     cardano-slotting,
     cborg,
     containers,
-    data-default-class,
```

Comment:
If you rebase against master, this change should disappear, since it's already there